lib/src/player/player_service.dart

```diff
@@ -307,7 +307,7 @@ class PlayerService {
     nextAudio = queue.$2.elementAt(randomIndex);
   }
 
-  Future<void> playPrevious() async {
+  Future<void> playPrevious({bool doubleTap = false}) async {
```

Comment:
what is this flag currently used for?

Reply:
yeah no, just do one change after the othere, no need to place some lines in advance :)